cves/CVE-2017-14170.yml

```diff
@@ -64,7 +64,19 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A missing End of File (EOF) check in FFmpeg's Material eXchange Format (mxf)
+  demuxer could cause a Denial of Service attack via a specifically crafted
```

Comment:
I don't think the target audience would know what demuxing is.

Reply:
Yeah, that's a really good point. I've update the description